packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -41,6 +45,36 @@ import { OdspDocumentService } from "./odspDocumentService";
 import { INewFileInfo, getOdspResolvedUrl, createOdspLogger, toInstrumentedOdspTokenFetcher } from "./odspUtils";
 import { createNewFluidFile } from "./createFile";
 
+// Please update the README file in odsp-driver-definitions if you change the defaultCacheExpiryTimeoutMs.
+const defaultCacheExpiryTimeoutMs: number = 2 * 24 * 60 * 60 * 1000;
+
+export const defaultStoragePolicy: Required<IDocumentStorageServicePolicies> = {
+    // By default, ODSP tells the container not to prefetch/cache.
```

Comment:
I do not think this comment is right. I think driver tells Loader not to cache, because driver does its own caching.
"ODSP" in your comment maybe read as ODSP service, and it's not the case - it's driver implementation that makes that choice.

Also, similar comment to maximumCacheDurationMs as in previous comment - I think it only applies to "live" service implementation, it should be infinite for LocalOdspDocumentServiceFactory

Reply:
It looks like it was already that way. So maybe open a bug to follow up if it's too hard to fix it?